SQL/Cleanup_patches/2018-08-20_remove_duplicate_incoming_path_from_config_tables.sql

```diff
@@ -0,0 +1,8 @@
+---
+--- Removing the unused IncomingPath configuration setting
+---
+
+DELETE FROM Config
+  WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='IncomingPath');
```

Comment:
I'm wondering if we should just have a cascading foreign key on the config table to make this unnecessary.